app/javascript/hooks/mutations/admin/roles/useDeleteRole.jsx

```diff
@@ -12,7 +12,13 @@ export default function useDeleteRole({ role, onSettled }) {
   return useMutation(
     () => axios.delete(`/admin/roles/${role.id}.json`),
     {
-      onError: () => { toast.error(t('toast.error.problem_completing_action')); },
+      onError: (error) => {
+        if (error.response?.status === 500) {
```

Comment:
That way you can explicity check for a `403` here

Reply:
Should not have to check for a 403 here since we are removing the "Delete Role" button on frontend for User, Guest and Admin.